code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -520,6 +529,7 @@
 	id = "poisonberryjuice"
 	description = "A tasty juice blended from various kinds of very deadly and toxic berries."
 	taste_description = "berries"
+	sanity_gain_ingest = 0 //bad for you
```

Comment:
It is "tasty" according to description. Should give sanity, while also being very bad for you in effects.

People eat deadly mushrooms in real life only to cleanse their stomachs, just because they enjoy the taste.

Reply:
Fair enough